### PR TITLE
Led subsystem2

### LINKED ISSUE #4: First Pass - LED Subsystem
Expected funtionality:

LED Subsystem
- When disabled (Higher in list is higher priority)
  - No light for no code
  - Red light for no comms
  - Orange light for gyro zeroed since code start
  - Green light for comms
- When enabled
  - Green light for beam-break detecting note
  - Orange light for default operation

### PR DESCRIPTION
**Pull request by Dino-MAJ**
Approve this because I actually made it good this time unlike last time when cohen merged me wrong <:(

### CODE REVIEW

File: src/main/cpp/Subsystems/LEDSubystem.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include "Subsystems/LEDSubsystem.h"
+#include "rev/CANSparkMax.h"
```
Comment (cpastyrn3357) [inline src/main/cpp/Subsystems/LEDSubystem.cpp]: Remove CANSparkMax include

File: src/main/cpp/Subsystems/LEDSubystem.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#include <frc/DriverStation.h>
+#include <frc/smartdashboard/SmartDashboard.h>
```
Comment (cpastyrn3357) [inline src/main/cpp/Subsystems/LEDSubystem.cpp]: Remove smartdashboard include

File: src/main/include/Subsystems/LEDSubsystem.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <frc2/command/CommandPtr.h>
```
Comment (cpastyrn3357) [inline src/main/include/Subsystems/LEDSubsystem.h]: Remove CommandPtr include

**Review by cpastyrn3357 — CHANGES_REQUESTED** [review]

**Review by cpastyrn3357 — APPROVED** [review]

**Comment by Dino-MAJ** [comment]
LESS GO THE KING IS BAKC AND READY TO RUMMBBBLLKLE
IMG_0499

**Pull request merged**